panoptes_aggregation/extractors/utilities.py

```diff
@@ -57,9 +58,93 @@ def pluck_fields(classification, pluck_keys):
             last_value = classification
             for keyi in key_path:
                 last_value = last_value[keyi.strip()]
-
-            answers["pluck." + key.strip()] = last_value
+            if(key == 'feedback'):
```

Comment:
Update the docstring for the function to indicate the `feedback` is a reserved key with a specific use.